web/modules/custom/solution/src/Plugin/Block/SolutionContentBlock.php

```diff
@@ -51,12 +55,12 @@ class SolutionContentBlock extends BlockBase implements ContainerFactoryPluginIn
    * @param string $plugin_definition
    *   The plugin implementation definition.
    */
-  public function __construct(array $configuration, $plugin_id, $plugin_definition, CurrentRouteMatch $current_route_match, EntityManager $entity_manager
-  ) {
+  public function __construct(array $configuration, $plugin_id, $plugin_definition, CurrentRouteMatch $current_route_match, EntityManager $entity_manager, ContextProviderInterface $solution_context) {
     parent::__construct($configuration, $plugin_id, $plugin_definition);
```

Comment:
Why the change to use a context, as the block access is determined by the route?